src/sentry/static/sentry/app/views/organizationProjectsDashboard/projectNav.jsx

```diff
@@ -36,8 +37,22 @@ const ProjectNav = createReactClass({
         title: t('Team'),
         to: `/organizations/${org.slug}/teams/new/`,
         onSelect: () => {
-          openCreateTeamModal({
+          // openCreateTeamModal({
+          //   organization: org,
+          //   onClose: data => {
+          //     if (!data) return;
+
+          //     browserHistory.push(`/settings/${org.slug}/teams/${data.slug}/members/`);
+          //   },
+          // });
```

Comment:
Could this be deleted instead of commented out?

Reply:
Hey Mark- it's a temporary quick & dirty branch for dev collaboration only. It won't ever be merged. Thanks for hopping on though.

Reply:
> It won't ever be merged.

Did this require a PR? What if we just pushed the branch?

Reply:
next time def